### PR TITLE
Move UtilsFromSpark to init.sc.

### PR DESCRIPTION
**Pull request by meh-ninja**
They're all the same and no need to multiply classes.
@vidma @andypetrella ?

### CODE REVIEW

**Comment by andypetrella** [comment]
Why not!

Is this safe when running on cluster? regarding closures and serialization things?

Also I saw that one test is failing, but it's not related to the changes ... :-S

**Comment by meh-ninja** [comment]
Tested on mesos, yes. It eliminates one class from `notebook` package - the `Utils` class, nothing more.
Failing test related to the scala interpreter, passed when retrying.

**Comment by andypetrella** [comment]
:+1:

**Comment by vidma** [comment]
> Failing test related to the scala interpreter, passed when retrying.

what do you mean by "passed when retrying"?

File: conf/scripts/init.sc [hunk 1]
```diff
@@ -24,7 +27,7 @@ import org.apache.spark.rdd._
     val envJars = sys.env.get("ADD_JARS")
     val propJars = sys.props.get("spark.jars").flatMap { p => if (p == "") None else Some(p) }
     val jars = List(propJars, envJars).collect{case Some(j) => j}.mkString(",")
-    notebook.Utils.resolveURIs(jars).split(",").filter(_.nonEmpty)
+    resolveURIs(jars).split(",").filter(_.nonEmpty)
```
Comment (vidma) [inline conf/scripts/init.sc]: I would worry about initialization order here.

`resolveURIs` uses `isWindows` & co. `val`s which are defined later than being used, and this happens inside of an initializer of `new java.io.Serializable {` class. I'd except for trouble :trollface:

**Comment by vidma** [comment]
good observation, but see my comment :trollface:

**Comment by vidma** [comment]
this should be quite serious, I'll make a fix PR soon.

File: conf/scripts/init.sc (hunk 1 above)
Comment (meh-ninja) [inline conf/scripts/init.sc]:
```
scala> globalScope.notAvailFn()
warning: there were 1 feature warning(s); re-run with -feature for details
res0: String = this is fun
```

**Comment by meh-ninja** [comment]
I agree there should be no randomly passing tests. It requires more time understanding what the presentationcompiler does in the first place. Not even mentioning all the problems with around running 2 of them at the same time...

File: conf/scripts/init.sc (hunk 1 above)
Comment (vidma) [inline conf/scripts/init.sc]: yes, the thing is about initialization order. so it's impossible to initialise/compute value of `val trollMe`at initialization time (as vals are initialized in order).

making it `def or lazy` would still work...

**Pull request merged**